frontend/src/helpers/index.ts

```diff
@@ -277,6 +302,8 @@ const exportFunctions = {
   storeClusterSettings,
   loadClusterSettings,
   getHeadlampAPIHeaders,
+  storeTableSettings,
```

Comment:
This commit and the previous commit has a lot of same changes, maybe squash them??

Reply:
I think you mean the previous 2 commits. I want to have them separated in case we end up reverting parts of this.